controllers/grid/settings/sections/form/PKPSectionForm.php

```diff
@@ -138,22 +139,37 @@ public function fetch($request, $template = null, $display = false)
 
     public function initData()
     {
+        $assignedSubeditors = [];
+        $subeditorUserGroups = [];
+
         if ($this->getSection() !== null) {
-            $this->setData([
-                'assignedSubeditors' => Repo::user()
-                    ->getCollector()
-                    ->filterByContextIds([Application::get()->getRequest()->getContext()->getId()])
-                    ->filterByRoleIds($this->assignableRoles)
-                    ->assignedToSectionIds([$this->getSectionId()])
-                    ->getIds()
-                    ->toArray(),
-            ]);
-        } else {
-            $this->setData([
-                'assignedSubeditors' => [],
-            ]);
+            $contextId = Application::get()->getRequest()->getContext()->getId();
+            $assignedSubeditors = Repo::user()
```

Comment:
Is `$assignedSubeditors` made redundant by `$subeditorUserGroups`? My understanding is that `$assignedSubeditors` was only used to check whether the user had an assignment to the section, but that it was missing the potential for one user's assignment to multiple user groups. Now that `$subeditorUserGroups` handles that properly, can we drop `$assignedSubeditors`?

Reply:
Thats better, done .